client/processes_tab.cpp

```diff
@@ -0,0 +1,12 @@
+#include "processes_tab.h"
+#include <QLabel>
+Processes_tab::Processes_tab(QWidget* parent)
+  : QWidget(parent)
+{
+    tab_widget = new QWidget();
```

Comment:
This widget is created as container for class content. But this class itself is derived class from `QWidget`. It may be used as working element of some layout.